src/overmap.cpp

```diff
@@ -2370,9 +2370,10 @@ void overmap::move_hordes()
             if(add_to_group == NULL) {
                 mongroup m(GROUP_ZOMBIE, p.x, p.y, p.z, 0, 1);
```

Comment:
It only uses the population if the monsters list is empty. Yeah, I can see why that's confusing, but not sure how to fix without subclassing or something.